src/type-system.ts

```diff
@@ -744,6 +745,18 @@ t.Date = ElysiaType.Date
 
 t.UnionEnum = ElysiaType.UnionEnum
 
+function getOrSetType<TSchema = unknown, TReturn = unknown>(
```

Comment:
This method implementation is based on [sinclare docs](https://github.com/sinclairzx81/typebox?tab=readme-ov-file#typeregistry-1) and [TypeSystem.Type](https://github.com/sinclairzx81/typebox/blob/6d54d12420ee8badaa4cb2a3492af4cba214eb95/src/system/system.ts#L55-L59).

It's not [recommendet anymore](https://github.com/sinclairzx81/typebox/issues/1049#issuecomment-2443242874) to use `TypeSystem` and instead switch to `TypeRegistry` and `FormatRegistry`.

This approach doesn't throw any `TypeSystemDuplicateTypeKind`.